org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -580,7 +586,11 @@ public String getFormatterProfileName() {
 		return formatterProfileName;
 	}
 
-	public MemberSortOrder getMemberSortOrder() {
+	/**
+	 * @deprecated - use IEclipsePreferences instead.
+	 */
+	@Deprecated
+	public String getMemberSortOrder() {
```

Comment:
it's useless to deprecate since the return type changed, so all clients would be potentially broken anyway. 
Since this is not API anyway, you can simply delete this method